passive/HUNT - Debug & Logic Parameters.py

```diff
@@ -0,0 +1,43 @@
+import re
+from org.zaproxy.zap.extension.script import ScriptVars
+
+''' find posible Server Side Template Injection using Hunt Methodology - https://github.com/bugcrowd/HUNT'''
```

Comment:
Should this be customized for this script? (vs. being the same as the SSTI one)

Reply:
Yes, it should be customized.